core/src/com/unciv/logic/multiplayer/Multiplayer.kt

```diff
@@ -0,0 +1,57 @@
+package com.unciv.logic.multiplayer
+
+import com.unciv.logic.GameInfo
+import com.unciv.logic.GameInfoPreview
+import com.unciv.logic.GameSaver
+import com.unciv.ui.saves.Gzip
+import java.util.*
+
+interface IFileStorage {
+    fun saveFileData(fileName: String, data: String)
+    fun loadFileData(fileName: String): String
+    fun getFileMetaData(fileName: String): IFileMetaData
+    fun deleteFile(fileName: String)
+}
+
+interface IFileMetaData {
+    fun getFileName(): String?
+    fun getLastModified(): Date?
+}
+
+class FileStorageConflictException: Exception()
```

Comment:
Introduced IFileMetaData to match FileStorage scheme
Changed DropBoxFileConflictException to FileStorageConflictException to match FileStorage scheme